web-app/src/services/tournaments.ts

```diff
@@ -0,0 +1,387 @@
+  };
+}
+
+/**
+ * Get the icon for a tournament.
+ *
+ * @export
+ * @param {string} name
+ * @returns {string}
+ */
+export function getTournamentIcon(name: string): string {
+  let icon = '';
+
+  switch (name) {
+    case 'futsal':
+      icon = 'far fa-futbol';
+      break;
+    case 'carrom-board':
+      icon = 'fas fa-vector-square';
+      break;
+    case 'table-tennis':
+      icon = 'fas fa-table-tennis';
+      break;
+    case 'chess':
+      icon = 'fas fa-chess';
+      break;
+    default:
+      icon = 'fas fa-gamepad';
+      break;
+  }
+
+  return icon;
+}
+
+/**
+ * Get list of tournament navigation routes.
+ *
+ * @export
+ * @param tournament
+ * @param params
+ * @returns {TournamentNavigationRouteInterface[]}
+ */
+export function getNavigationRoutes(
+  tournament: TournamentDataInterface,
+  params: { game: string, season: string }
+): TournamentNavigationRouteInterface[] {
+  const routes: TournamentNavigationRouteInterface[] = [];
+  const { game, season } = params;
+  const subRoutes = sortBy(SUB_ROUTES, 'sortOrder') as any;
+
+  const hasPoints = (tournament.points && tournament.points.length > 1);
+  const hasKnockouts = (tournament.rounds && tournament.rounds.length > 1);
+  const hasStats = (tournament.stats && Object.values(tournament.stats).length);
+
+  Object.values(subRoutes).forEach((subRoute: any) => {
+    const { path, name } = subRoute;
+
+    // Avoid inclusion of the navigations which do not have any data.
+    if (
+      (path === SUB_ROUTES.STATS.path && !hasStats) ||
+      (path === SUB_ROUTES.POINTS.path && !hasPoints) ||
+      (path === SUB_ROUTES.KNOCKOUTS.path && !hasKnockouts)
+    ) {
+      return;
+    }
+
+    routes.push({
+      name,
+      path: `/${game}/${season}/${path}`
+    });
+  });
+
+  return routes;
+}
+
+/**
+ * Get list of all tournaments.
+ *
+ * @param {TournamentInterface[]} tournaments
+ * @returns {SeasonInterface[]}
+ */
+function getAllSeasons(tournaments: TournamentInterface[]): SeasonInterface[] {
+  let seasons: SeasonInterface[] = [];
+
+  tournaments.forEach(tournament => {
+    tournament.seasons.forEach(season => {
+      season.parentTournament = {
+        id: tournament.id,
+        name: tournament.name,
+        shortName: tournament.shortName
+      };
+    });
+
+    seasons = seasons.concat(tournament.seasons);
+  });
+
+  seasons.sort((a, b) => differenceInDays(new Date(a.startDate), new Date(b.startDate))).reverse();
+
+  return seasons;
+}
+
+/**
+ * Get non-played fixtures of a tournament.
+ *
+ * @export
+ * @param {FixtureInterface[]} fixturesList
+ * @param {number} [limit=0]
+ * @returns {FixtureInterface[]}
+ */
+function getFixtures(fixturesList: FixtureInterface[], limit: number = 0): FixtureInterface[] {
+  let fixtures = fixturesList
+    .filter(fixture => ['played', 'forfeited'].indexOf(fixture.status.toLowerCase()) < 0)
+    .sort((a, b) => {
+      return compareAsc(a.date, b.date);
```

Comment:
```suggestion
       compareAsc(a.date, b.date);
```
Let's remove the curly braces and the return statement as well